engine/src/main/java/org/pentaho/di/trans/steps/textfileoutput/TextFileOutput.java

```diff
@@ -1034,6 +1019,28 @@ private void createParentFolder( String filename ) throws Exception {
     }
   }
 
+  boolean isWriteEnclosureForFieldName( ValueMetaInterface v, String fieldName ) {
```

Comment:
You'll need javadoc comment for all new functions.

- isWriteEnclosureForFieldName
- isWriteEnclosureForValueMetaInterface
- isWriteEnclosureForWriteField
- isWriteEnclosed

Reply:
Added javadoc comments in all these methods